Sources/Core/OpaquePointer.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+// This `OpaquePointer` implementation is known to crash 5.4 and 5.5 compiler releases on Windows.
```

Comment:
Thanks for the comment, that is a good idea to record!

Reply:
The formatter I use has done a stupid (in my opinion) thing by indenting everything within `#if` block. I've pushed one more time to fix that.